<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minesweepr Plus</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 rgba(45, 45, 45, 255), stop:1 rgba(113, 113, 113, 255))</string>
  </property>
  <widget class="QLineEdit" name="UsernameField">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>160</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(236, 158, 255, 100);
border-radius:3px;
color:white;
</string>
   </property>
   <property name="placeholderText">
    <string> username...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordField">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>209</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(236, 158, 255, 100);
border-radius:3px;
color: white;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string> password...</string>
   </property>
  </widget>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>80</y>
     <width>151</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
background-color:rgba(0,0,0,0)</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorLabel">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>144</y>
     <width>291</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
color:rgb(255, 82, 82)
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="SignupButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton::hover
{
background-color:rgb(216, 140, 234);
}

QPushButton
{
background-color:rgba(236, 158, 255, 170);
border-radius:3px;
color: white;
};</string>
   </property>
   <property name="text">
    <string>register</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OldUserButton">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>289</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
color:white;</string>
   </property>
   <property name="text">
    <string>Have An Account? Login Now!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ConfirmPasswordField">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>258</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(236, 158, 255, 100);
border-radius:3px;
color: white;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string> repeat password...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
